autogpt/configurator.py

```diff
@@ -82,10 +82,22 @@ def create_config(
 
     if gpt3only:
         logger.typewriter_log("GPT3.5 Only Mode: ", Fore.GREEN, "ENABLED")
+        CFG.set_smart_token_limit(CFG.fast_token_limit)
```

Comment:
This feels unexpected to me. Suppose I'm experimenting and have changed my env.SMART_TOKEN_LIMIT, I'll wonder why it's not being picked up.

I'd understand if it used the value from my env and errored out, or (begrudgingly) if it used the default, known GPT-3.5 limit (independently of what was in the .env) - but with this second option, I'd wonder how to override the default. The natural thing for overriding the default would be to change the env.SMART_TOKEN_LIMIT.